pkg/bot/socketslack.go

```diff
@@ -338,8 +343,14 @@ func (b *SocketSlack) send(event socketSlackMessage, req string, resp interactiv
 	}
 
 	// Upload message as a file if too long
+	var file *slack.File
+	var err error
 	if len(markdown) >= slackMaxMessageSize {
-		return uploadFileToSlack(event.Channel, resp, b.client)
+		file, err = uploadFileToSlack(event.Channel, resp, b.client)
+		if err != nil {
+			return err
+		}
+		resp.IgnoreOriginalResponse = true
```

Comment:
Personally I don't like that we introduce the new bool property just to "talk" between 2 layers. IMO we can just do that:

```suggestion
		// The message sections were already sent, we only need to
		// take care of sending inputs it there are any.
		resp = interactive.Message{
			Inputs: resp.Inputs,
		}
```

It's simple and says already what we want to achieve. Then the renderer doesn't need to be aware of what to do on `IgnoreOriginalResponse`.

Reply:
I am aware of that, but there are cases we have both sections and inputs. So, I cannot decide on something based on the component inside the interactive message.

Reply:
Now I understand your previous approach 👍 Thanks for clarification 👍